php/class-plugin.php

```diff
@@ -163,6 +169,9 @@ function init() {
 		if ( $this->is_module_active( 'deferred_customize_widgets' ) ) {
 			$this->deferred_customize_widgets = new Deferred_Customize_Widgets( $this );
 		}
+		if ( $this->is_module_active( 'customize_settings_snapshot' ) ) {
+			$this->customize_settings_snapshot = new Customize_Settings_Snapshot( $this );
```

Comment:
@valendesigns Given how the transaction patch is basically two parts, the extensions to the `WP_Customize_Manager` and then a separate `WP_Customize_Transaction` that is instantiated by the former, maybe it would make sense to take the same approach here, where we can have a `Customize_Snapshot_Manager` (or `Customize_Snapshots` or `Customize_Settings_Snapshotting`?) and a `Customize_Settings_Snapshot`. This would allow our code here in the plugin to more closely mirror the transactions patch, and make it easier to unit-test since we could more easily instantiate `Customize_Settings_Snapshot` in isolation.